### PR TITLE
[1LP][RFR] Check pagination on configuration pages

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
{{pytest: cfme/tests/configure/test_paginator.py }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 263991fa61ee1ba6da2aafd67ff5621ff3c9b146:

`cfme/tests/configure/test_paginator.py`:
- :bangbang: Line 26:8: **E131** *continuation line unaligned for hanging indent*
- :bangbang: Line 30:9: **E131** *continuation line unaligned for hanging indent*
- :bangbang: Line 30:9: **E265** *block comment should start with '# '*
- :bangbang: Line 33:9: **E131** *continuation line unaligned for hanging indent*
- :bangbang: Line 33:9: **E265** *block comment should start with '# '*
- :bangbang: Line 55:101: **E501** *line too long (115 > 100 characters)*
- :bangbang: Line 67:9: **E265** *block comment should start with '# '*
- :heavy_exclamation_mark: Line 102:39: **W292** *no newline at end of file*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3259d29b4ab8fe4e4505b586003ae52b931c4d9c:

`cfme/tests/configure/test_paginator.py`:
- :red_circle: Line 5:1: **F401** *'cfme.configure.configuration.analysis_profile.AnalysisProfileDetailsView' imported but unused*
- :bangbang: Line 5:101: **E501** *line too long (101 > 100 characters)*
- :bangbang: Line 74:1: **E302** *expected 2 blank lines, found 1*
- :bangbang: Line 81:1: **E302** *expected 2 blank lines, found 1*
- :bangbang: Line 116:40: **E231** *missing whitespace after ','*
- :heavy_exclamation_mark: Line 119:71: **W292** *no newline at end of file*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0a38b779b62fa2eb47683747870dfc6b5ef861e3:

`cfme/tests/configure/test_paginator.py`:
- :red_circle: Line 6:1: **F401** *'cfme.configure.configuration.analysis_profile.AnalysisProfileDetailsView' imported but unused*
- :bangbang: Line 6:101: **E501** *line too long (101 > 100 characters)*
- :heavy_exclamation_mark: Line 124:71: **W292** *no newline at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8b55ce38c1ccb0ea21846d46309394a22622a32e:

`cfme/tests/configure/test_paginator.py`:
- :red_circle: Line 8:1: **F401** *'cfme.configure.configuration.diagnostics_settings.ZoneCollectLog' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0631393e0800d2c46d509952f29c6422763be582:

`cfme/tests/configure/test_paginator.py`:
- :red_circle: Line 95:17: **F821** *undefined name 'BZ'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/configure/test_paginator.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+@pytest.mark.parametrize('place_info', general_list_pages,
+                         ids=['{}_{}'.format(set_type[0], set_type[2].lower())
+                              for set_type in general_list_pages])
+def test_paginator(appliance, place_info):
+    """
+        Check paginator is visible for config pages
+    """
+    place_name, place_class, place_navigation, paginator_expected_result = place_info
+    if appliance.version < '5.9' and place_navigation == 'HelpMenu':
+        pytest.skip('No Help Menu tab for 5.8 version')
```
Comment (quarckster) [inline cfme/tests/configure/test_paginator.py]: **OPTIONAL**
It can be uncollected.

Comment (psav) [reply cfme/tests/configure/test_paginator.py]: **REQUIRED:** I'd like an uncollect here, this test will never be expected to pass.

**Comment by rlbabyuk** [comment]
PRT for last commit 22631, for failed test cases in 5.9 there is a bz 1560949. Didn't add in tests as this should be fixed in next build

**Pull request merged**